AppCenter/AppCenter/Internals/Ingestion/MSOneCollectorIngestion.m

```diff
@@ -94,6 +97,38 @@ - (NSURLRequest *)createRequest:(NSObject *)data
                     stringWithFormat:@"%lld",
                                      (long long)[MSUtility nowInMilliseconds]]
          forKey:kMSOneCollectorUploadTimeKey];
+
+  // Gather tokens from logs.
+  NSMutableString *ticketKeyString = [NSMutableString new];
+  for (id<MSLog> log in container.logs) {
+    MSCommonSchemaLog *csLog = (MSCommonSchemaLog *)log;
+    if (csLog.ext.protocolExt) {
+      NSArray<NSString *> *ticketKeys = [[[csLog ext] protocolExt] ticketKeys];
+      for (NSString *ticketKey in ticketKeys) {
+        NSString *token = [[MSTicketCache sharedInstance] ticketFor:ticketKey];
+        if (token) {
+
+          /*
+           * Format to look like this:
+           * "ticketKey1"="d:token1";"ticketKey2"="d:token2" or
+           * "ticketKey1"="p:token1";"ticketKey2"="p:token2". The value (p: vs.
```

Comment:
We use the modern json format for One Collector in Android. This is the backward compatible format from Vortex.